src/components/ChatbotUi.js

```diff
@@ -77,7 +78,7 @@ const ChatbotUi = ({ template }) => {
   return (
     <StyledChatBotUiContainer data-testid="chatbot-ui" template={template}>
       {/* We wrapped this in a Suspense. We can use this opportunity to render a loading state if we decided we want that */}
-      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} width={771} height={82} />}>
+      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} height={82} />}>
         <LazyChatbot serverBaseUrl={CHATBOT_SERVER_BASE_URL} />
```

Comment:
```suggestion
        <LazyChatbot
          serverBaseUrl={CHATBOT_SERVER_BASE_URL}
          suggestedPrompts={[
            "How do you deploy a free cluster in Atlas?",
            "How do you import or migrate data into MongoDB Atlas?",
            "Get started with MongoDB",
            "Why should I use Atlas Search?",
          ]}
        />
```

We need to define the suggested prompts client side